orientation-sensor/idlharness.https.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Orientation Sensor IDL tests</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/orientation-sensor/">
+<link rel="help" href="https://w3c.github.io/sensors/">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/idlharness.js"></script>
+<style>
+  pre {
+    display: none;
+  }
+</style>
+<div id="log"></div>
+
+<pre id="idl">
+interface Event {
+};
+
+interface Error {
+};
+
+dictionary EventInit {
+};
+
+interface EventTarget {
+};
+
+interface EventHandler {
+};
+</pre>
+
+<pre id="generic-idl">
+[SecureContext]
+interface Sensor : EventTarget {
+  readonly attribute boolean activated;
+  readonly attribute DOMHighResTimeStamp? timestamp;
+  void start();
+  void stop();
+  attribute EventHandler onchange;
+  attribute EventHandler onactivate;
+  attribute EventHandler onerror;
+};
+
+dictionary SensorOptions {
+  double? frequency;
+};
+
+[SecureContext, Constructor(DOMString type, SensorErrorEventInit errorEventInitDict)]
+interface SensorErrorEvent : Event {
+  readonly attribute Error error;
+};
+
+dictionary SensorErrorEventInit : EventInit {
+  required Error error;
+};
+</pre>
+
+<pre id="orientation-idl">
+typedef (Float32Array or Float64Array or DOMMatrix) RotationMatrixType;
+interface OrientationSensor : Sensor {
+  readonly attribute FrozenArray<double>? quaternion;
+  void populateMatrix(RotationMatrixType targetMatrix);
+};
+
+[Constructor(optional SensorOptions sensorOptions)]
+interface AbsoluteOrientationSensor : OrientationSensor {
+};
+</pre>
+
+<script>
+
+(() => {
```

Comment:
OK, so I checked on irc ans the answer is: _"Has to work in latest Chrome, Fx, Safari, Edge"_.

So you're good.